File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/billing/shared/dto/bill-new-settlement.dto.ts
import { EmployeeCashTransaction } from "../billing-transaction.model";
import { PatientCreditInvoices_DTO } from "./bill-credit-invoice-details.dto";

export class BillNewSettlement_DTO {
    public SettlementId: number = 0;
    public SettlementType: string = "";
    public PatientId: number = 0;
    public PayableAmount: number = 0;
    public RefundableAmount: number = 0;
    public PaidAmount: number = 0;
    public ReturnedAmount: number = 0;
    public DepositDeducted: number = 0;
    public DueAmount: number = 0;
    public DiscountAmount: number = 0;
    public PaymentMode: string = "";
    public PaymentDetails: string = "";
    public CounterId: number = 0;
    public Remarks: string = "";
    public BillingTransactions = new Array<PatientCreditInvoices_DTO>();
    public PHRMInvoiceTransactionModels = new Array<PatientCreditInvoices_DTO>();
    public CollectionFromReceivable: number = 0;
    public DiscountReturnAmount: number = 0;
    public OrganizationId: number = null;
    public StoreId: number = null;
    public BillingUser: string = "";
    public BillReturnIdsCSV = new Array<any>();
    public empCashTransactionModel = new Array<EmployeeCashTransaction>();
}

File: Code/Websites/DanpheEMR/Services/Billing/DTO/BillNewSettlement_DTO.cs
using DanpheEMR.ServerModel;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System;

namespace DanpheEMR.Services.Billing.DTO
{
    public class BillNewSettlement_DTO
    {
        public int SettlementId { get; set; }
        public string SettlementType { get; set; }
        public int PatientId { get; set; }
        public double PayableAmount { get; set; }
        public double RefundableAmount { get; set; }
        public double PaidAmount { get; set; }
        public double ReturnedAmount { get; set; }
        public double DepositDeducted { get; set; }
        public double DueAmount { get; set; }
        public double DiscountAmount { get; set; }
        public string PaymentMode { get; set; }
        public string PaymentDetails { get; set; }
        public int CounterId { get; set; }
        public string Remarks { get; set; }
        public List<PatientCreditInvoices_DTO> BillingTransactions { get; set; }
        public List<PatientCreditInvoices_DTO> PHRMInvoiceTransactionModels { get; set; }
        public PatientModel Patient { get; set; }
        public double CollectionFromReceivable { get; set; }
        public double DiscountReturnAmount { get; set; }
        public int? OrganizationId { get; set; }
        public int? StoreId { get; set; }
        public string BillingUser { get; set; }
        public List<int> BillReturnIdsCSV { get; set; }
        public List<EmpCashTransactionModel> empCashTransactionModel { get; set; }
    }
}
